etc/skel/.config/hypr/hyprland.conf

```diff
@@ -1,463 +1,22 @@
-windowrulev2 = pin, title:^(danmufloat)$
-windowrulev2 = rounding 5, title:^(danmufloat|termfloat)$
-windowrulev2 = animation slide right, class:^(kitty|Alacritty)$
-windowrulev2 = noblur, class:^(org.mozilla.firefox)$
-# Decorations related to floating windows on workspaces 1 to 10
-windowrulev2 = bordersize 2, floating:1, onworkspace:w[fv1-10]
-windowrulev2 = bordercolor $cachylblue, floating:1, onworkspace:w[fv1-10]
-windowrulev2 = rounding 8, floating:1, onworkspace:w[fv1-10]
-# Decorations related to tiling windows on workspaces 1 to 10
-windowrulev2 = bordersize 3, floating:0, onworkspace:f[1-10]
-windowrulev2 = rounding 4, floating:0, onworkspace:f[1-10]
-# Windows Rules End #
-
-# Workspaces Rules https://wiki.hyprland.org/0.45.0/Configuring/Workspace-Rules/ #
-# workspace = 1, default:true, monitor:$priMon
-# workspace = 6, default:true, monitor:$secMon
-# Workspace selectors https://wiki.hyprland.org/0.45.0/Configuring/Workspace-Rules/#workspace-selectors
-# workspace = r[1-5], monitor:$priMon
-# workspace = r[6-10], monitor:$secMon
-# workspace = special:scratchpad, on-created-empty:$applauncher
-# no_gaps_when_only deprecated instead workspaces rules with selectors can do the same
-# Smart gaps from 0.45.0 https://wiki.hyprland.org/0.45.0/Configuring/Workspace-Rules/#smart-gaps
-workspace = w[tv1-10], gapsout:5, gapsin:3
-workspace = f[1], gapsout:5, gapsin:3
-# Workspaces Rules End #
-
-# Layers Rules #
-layerrule = animation slide top, logout_dialog
-# layerrule = animation popin 50%, waybar
-layerrule = animation slide down, waybar
-layerrule = animation fade 50%, wallpaper
-# Layers Rules End #
+$mainMod = SUPER
 
-# Environment Variables #
-envd = HYPRCURSOR_THEME,Bibata-Modern-Classic
-envd = HYPRCURSOR_SIZE,24
-envd = XCURSOR_SIZE,24
-envd = QT_CURSOR_THEME,Bibata-Modern-Classic
-envd = QT_CURSOR_SIZE,24
-# Environment Variables End #
+# ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+# ┃                         Source Files                        ┃
+# ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+source = /usr/share/cachyos-hyprland-settings/animations.conf
+source = /usr/share/cachyos-hyprland-settings/autostart.conf
+source = /usr/share/cachyos-hyprland-settings/decorations.conf
+source = /usr/share/cachyos-hyprland-settings/enviroment.conf
+source = /usr/share/cachyos-hyprland-settings/input.conf
+source = /usr/share/cachyos-hyprland-settings/keybinds.conf
+source = /usr/share/cachyos-hyprland-settings/monitor.conf
+source = /usr/share/cachyos-hyprland-settings/variables.conf
+source = /usr/share/cachyos-hyprland-settings/windowrules.conf
+
+# If you want to add extra configs, please create a new .conf file and source it like this:
+# source = /usr/share/cachyos-hyprland-settings//custom.conf
+# If you want to use the current colors for something, please also source the "/usr/share/cachyos-hyprland-settings/colors.conf".
```

Comment:
With the addition of prepackaged configs being /usr, this needs to be changed now

```suggestion
# Modifying these configs can be done by creating a user defined config in the home directory, e.g.
## ~/.config/hyprland/conf.d/user-config.conf
# source ~/.config/hyprland/conf.d/user-config.conf
```